ionic/src/app/login/login.component.ts

```diff
@@ -54,11 +53,19 @@ export class LoginComponent implements OnInit {
   async addDevice() {
     try {
       let body = {};
-      body['chip_id'] = "chip1"
-      body['meta'] = { "name": "device" };
+      body['chip_id'] = "123456"
+      body['meta'] = {
+        "name": "Access Control",
```

Comment:
why is this hard coded?

Reply:
sir if i am not given  meta data hard coded  then we require a form  or we create j json file. this is also hard coded